src/security-rules/security-rules-api-client.ts

```diff
@@ -39,6 +39,11 @@ export interface RulesetResponse extends RulesetContent {
   readonly createTime: string;
 }
 
+export interface ListRulesetsResponse {
+  readonly rulesets: Array<{name: string, createTime: string}>;
```

Comment:
I assume the bracket syntax doesn't work here?

Reply:
Right. Linter prefers it this way.